entries/c/commuterbenefits.com.json

```diff
@@ -1,12 +1,11 @@
 {
   "CommuterBenefits": {
```

Comment:
Based on their lack of GDPR compliance I'd say they seem to only cater to the U.S. market.

Reply:
"Edenred" is international, but "Commuter Benefits" seems to be US only.